caml-font.el

```diff
@@ -299,6 +299,8 @@
          ((looking-at caml-font-newline-re)
           (goto-char (match-end 0))
           (setq continue (caml-font-put-state (match-end 0) (cons nil depth))))
+         ((caml-font-looking-at caml-font-ident-re)
```

Comment:
Is this useful?  Won't the [`caml-font-looking-at caml-font-ident-or-num-re`](https://github.com/ocaml/caml-mode/pull/4/files#diff-94c3dafca06021b3008d244c51ff466cL245) above already catch this case?

Reply:
The `caml-font-ident-or-num-re` above is in another state (outside comments).

Reply:
Thanks — did not see that when glancing at it in github.